creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -343,11 +343,12 @@ const _determineNewReplicaSetWhenTwoNodesAreUnhealthy = (
  */
 const _selectRandomReplicaSetNodes = async (
   healthyReplicaSet: Set<string>,
-  numberOfUnhealthyReplicas: number,
+  unhealthyReplicasSet: Set<string>,
```

Comment:
nit - i think the jsdoc above needs to change for both the param as well as the comment about "does not have any existing user state".

Reply:
fixed!